extension.js

```diff
@@ -128,7 +129,7 @@ function destroyOverlay() {
 function onHotkeyPressed() {
   Log.debug("Hot key pressed");
   //Clear the lock bit so the status of Caps_Lock won't affect the functionality
```

Comment:
Could you move this comment to utils.js so we know what ~2(caps_lock) and ~16(num_lock) are.